sdk/Dagger.SDK/Dagger.SDK.csproj

```diff
@@ -15,9 +15,9 @@
                 OutputItemType="Analyzer"
                 ReferenceOutputAssembly="false" />
     </ItemGroup>
-
+    
     <ItemGroup>
-        <AdditionalFiles Include="introspection.json"/>
+        <AdditionalFiles Include="introspection.json" />

```

Comment:
We need to keep this somehow? Or generator doesnt run code and when we delete file, `AdditionalFiles` is removed automatically it seems. So need to manually add this back in... not sure how we handle this.

Reply:
I think we should keep this. The generator requires this file to generate sdk. If the generator keep silence, the SDK will completely broken due to `Dagger` class requires during call `Connect`.

Reply:
How does this work in the world of Powershell?

1. We have source generator project

2. We have this `wrapper` project (it depend on introspection.json)

3. The generator need to generate code

4. How does package look for end user? 
- no introspection.json
- dagger develop -sdk dotnet (this this add introspection.json somehow?)

Reply:
> How does this work in the world of Powershell?
> 
> 1. We have source generator project
> 2. We have this `wrapper` project (it depend on introspection.json)
> 3. The generator need to generate code
> 4. How does package look for end user?
> 
> * no introspection.json
> * dagger develop -sdk dotnet (this this add introspection.json somehow?)

For Dagger module, the engine will inject introspection during generate code. You can see at https://github.com/wingyplus/dagger-dotnet-sdk/pull/7/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R102.